### DISCUSSION TITLE
Azure Monitor: Unable to render timeseries result from `make-series`

### DISCUSSION THREAD

**Discussion by sanderaernouts** [grafana/grafana#43265] [Data Sources]
**What happened**:
Using as Kusto query with the make-series function does not produce a graph,

**What you expected to happen**:
The make-series query should have shown a graph.

**How to reproduce it (as minimally and precisely as possible)**:
Run this query (from the examples):
```
let T = range Timestamp from $__timeFrom to ($__timeTo + -30m) step 1m
  | extend   Person = dynamic(["Torkel", "Daniel", "Kyle", "Sofia"])
  | extend   Place  = dynamic(["EU",     "EU",     "US",   "EU"])
  | mvexpand Person, Place
  | extend   HatInventory = rand(5)
  | project  Timestamp, tostring(Person), tostring(Place), HatInventory;

T | make-series avg(HatInventory) default=double(null) on Timestamp from $__timeFrom to $__timeTo step 1m by Person, Place
  | extend series_decompose_forecast(avg_HatInventory, 1) | project-away *residual, *baseline, *seasonal

```

**Screenshots**
![image](https://user-images.githubusercontent.com/6184673/146322728-e3718332-2796-46de-8c52-fa16e3f654f4.png)

**Anything else we need to know?**:
In table format the data looks like this:
![image](https://user-images.githubusercontent.com/6184673/146323242-f6b95a3e-cac4-44eb-9c2f-cc6ce8479181.png)

From the screenshot it seems that all columns are detected as string and the timestamp column is not recognized as time in this `make-series` format.

This is a copy of [issue 292](https://github.com/grafana/azure-data-explorer-datasource/issues/292) in the [Azure Data Explorer datasource](https://github.com/grafana/azure-data-explorer-datasource).

Both Azure Date Explorer and Azure monitor use the same query language so I would expect both data sources to support the `make-series` timeseries format. You can use `mv-expand` to expand the series back to a normal table format but that is quite slow in my experience.

**Environment**:
Grafana Cloud

**Comment by sunker** [comment]
Thanks for creating this issue @sanderaernouts. We currently don't support this time series format in the Azure Monitor data source. This looks like a feature request, so I'll convert it to a discussion.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
